[2022-12-06 14:56:20,866] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: my_mongo_db_dag.LoadPositionsInMongoDB 2022-12-06T14:56:13.829646+00:00 [queued]>
[2022-12-06 14:56:20,885] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: my_mongo_db_dag.LoadPositionsInMongoDB 2022-12-06T14:56:13.829646+00:00 [queued]>
[2022-12-06 14:56:20,886] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2022-12-06 14:56:20,886] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2022-12-06 14:56:20,886] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2022-12-06 14:56:20,897] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): LoadPositionsInMongoDB> on 2022-12-06T14:56:13.829646+00:00
[2022-12-06 14:56:20,901] {standard_task_runner.py:52} INFO - Started process 530 to run task
[2022-12-06 14:56:20,906] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_mongo_db_dag', 'LoadPositionsInMongoDB', '2022-12-06T14:56:13.829646+00:00', '--job-id', '4', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_mongo_db_dag.py', '--cfg-path', '/tmp/tmpqfyq7dkp', '--error-file', '/tmp/tmpqsvbscim']
[2022-12-06 14:56:20,906] {standard_task_runner.py:77} INFO - Job 4: Subtask LoadPositionsInMongoDB
[2022-12-06 14:56:20,958] {logging_mixin.py:104} INFO - Running <TaskInstance: my_mongo_db_dag.LoadPositionsInMongoDB 2022-12-06T14:56:13.829646+00:00 [running]> on host fd35ce852e5b
[2022-12-06 14:56:21,027] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_mongo_db_dag
AIRFLOW_CTX_TASK_ID=LoadPositionsInMongoDB
AIRFLOW_CTX_EXECUTION_DATE=2022-12-06T14:56:13.829646+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-06T14:56:13.829646+00:00
[2022-12-06 14:56:21,301] {logging_mixin.py:104} INFO - debut de chargement des données dans mongodb
[2022-12-06 14:56:21,596] {logging_mixin.py:104} INFO - fin de chargement des données dans mongodb
[2022-12-06 14:56:21,600] {python.py:151} INFO - Done. Returned value was: None
[2022-12-06 14:56:21,609] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=my_mongo_db_dag, task_id=LoadPositionsInMongoDB, execution_date=20221206T145613, start_date=20221206T145620, end_date=20221206T145621
[2022-12-06 14:56:21,630] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-12-06 14:56:21,639] {local_task_job.py:151} INFO - Task exited with return code 0
